build/scripts/cibuild.cmd

```diff
@@ -1,2 +1,7 @@
 @echo off
-powershell -noprofile -executionPolicy RemoteSigned -file "%~dp0\build.ps1" -cibuild -build -restore -bootstrap -binaryLog %*
+if "%1"=="-testBuildCorrectness" ( 
+    REM Temporary work around until the netci.groovy change takes
+    call %~dp0\test-build-correctness.cmd -cibuild -release
```

Comment:
This has to be kept until this PR is merged because the current state of netci.groovy depends on it. Once this is merged I will immediately follow up with a PR to delete this as it won't be needed anymore.